packages/components/package.json

```diff
@@ -12,7 +12,7 @@
     "build": "nps build",
     "clean": "nps clean",
     "cleanAll": "nps cleanAll",
-    "prepare": "yarn build",
+    "prepublishOnly": "yarn build",
```

Comment:
We may need to update the build steps in the LabKey UI Components project for TC after this change.  It is currently doing just an npmInstall  but no build (to avoid building twice) before running tests.

Reply:
Who do we talk to about that? Trey?

Reply:
Trey or Dan or I can do it, but the tests seem to have run just fine, so probably I'm wrong.